src/oas/transformers/schema/__tests__/schema.spec.ts

```diff
@@ -0,0 +1,112 @@
+import { translateSchemaObject } from '..';
+import { OASSchemaObject } from '../types';
+
+const translate = (schemaObject: OASSchemaObject) => translateSchemaObject(schemaObject, { pruneNotSupported: [] });
+
+describe('translateSchemaObject', () => {
```

Comment:
@philsturgeon I went a bit beyond the initial requirements we had in the ticket and applied some suggestions you left there.
It's not the newest draft, but it's also not Draft 6. :laughing: 
Would you like to take a look to see if there's something else you find useful covering in this converter?

@marbemac you may also want to look into this at your leisure.